openedx/features/idea/api_views.py

```diff
@@ -12,7 +12,7 @@ class FavoriteAPIView(APIView):
     """
     FavoriteAPIView is used to toggle favorite idea for the user
     """
-    authentication_classes = (SessionAuthentication, BasicAuthentication)
+    authentication_classes = (SessionAuthenticationAllowInactiveUser,)
```

Comment:
Is there any specific reason to remove `BasicAuthentication`? API won't be accessible through basic authentication if we remove this.